.github/workflows/build.yml

```diff
@@ -128,6 +128,8 @@ jobs:
       run: |
         echo "Running checks..."
         ./ietf/manage.py check
+        echo "Validating migrations..."
+        $do ./ietf/manage.py makemigrations | tee /dev/stderr | $do grep -q "^No changes detected$" || die "Model changes without migrations found."
```

Comment:
$do (in several places here) no longer makes sense?

Reply:
die is also locally defined in the source script.

Reply:
ok, I was wondering what $do was used for. I assumed it was some special magic bash keyword...
I rework the code.